src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/IInternalDebugLaunchProvider.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Microsoft.VisualStudio.ProjectSystem.Debug;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Debug;
+internal interface IInternalDebugLaunchProvider : IDebugLaunchProvider
```

Comment:
We generally version interfaces with numbers, rather than using names like "internal". So this would be `IDebugLaunchProvider2`.

Reply:
`IDebugLaunchProvider` is defined in CPS rather project-system, so it's better to reserve `IDebugLaunchProvider2` there?

Reply:
CPS doesn't have the concept of launch profiles. We can leave this as is.